kitsune/settings.py

```diff
@@ -1361,4 +1361,12 @@ def filter_exceptions(event, hint):
 WAGTAIL_SITE_NAME = config("WAGTAIL_SITE_NAME", default="Mozilla Support CMS")
 WAGTAILADMIN_BASE_URL = config("WAGTAILADMIN_BASE_URL", default="")
 WAGTAILIMAGES_MAX_UPLOAD_SIZE = IMAGE_MAX_FILESIZE
+WAGTAILADMIN_RICH_TEXT_EDITORS = {
+    "default": {
```

Comment:
should we use a different editor here like `sumo` in order to avoid overriding the default?

Reply:
Ah, I was looking at this like defining a sane default. It looks like the out-of-the-box defaults would include:
```

h2, h3, h4 - heading elements
bold, italic - bold / italic text
ol, ul - ordered / unordered lists
hr - horizontal rules
link - page, external and email links
document-link - links to documents
image - embedded images
embed - embedded media 
```

Which is actually pretty ok for us. So. I'll alter or remove this.